src/main/js/tansaction/transaction-service.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * Account service to abstract the logic to interact with the transaction API.
+ *
+ * @author Xiaobing Hou
+ * @date 06/04/2022
+ */
+
+import axios from "axios";
+
+const PATH = "/api/v1/transaction";
+const FIND_ALL_PATH = "findAll";
+const DEL_PATH = "delete";
+const EDIT_PATH = "modify";
+
+class TransactionService {
+  /**
+   * To get all transaction .
+   *
+   * @param bankAccountId
+   * @returns {Promise<AxiosResponse<any>>} Promise
+   * @author Xiaobing Hou
+   * @date 06/03/2022
+   */
+  findAllTransactionsByBankId = (bankAccountId) => {
+    return axios.get(`${PATH}/${FIND_ALL_PATH}/${bankAccountId}`);
+  };
+
+  /**
+   * To del transaction .
+   *
+   * @returns {Promise<AxiosResponse<any>>} Promise
+   * @author Xiaobing Hou
+   * @date 06/03/2022
+   * @param transactionId
+   * @param bankAccountId
+   */
+  deleteTranByTranIdAndBankId = (transactionId, bankAccountId) => {
+    return axios.delete(
+      `${PATH}/${DEL_PATH}/${transactionId}/${bankAccountId}`
+    );
+  };
+
+  /**
+   * To edit transaction .
+   *
+   * @returns {Promise<AxiosResponse<any>>} Promise
+   * @author Xiaobing Hou
+   * @date 06/03/2022
+   * @param transactionId
+   * @param values
+   */
```

Comment:
Can we please update all the doc comments to follow this common format? Not sure why have return at the top here.

```JavaScript
 /**
   * To get all transaction.
   *
   * @param bankAccountId Bank account id
   * @returns {Promise<AxiosResponse<any>>} Promise
   * @author Xiaobing Hou
   * @date 06/03/2022
   */
```

Reply:
done